infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -135,7 +145,28 @@ export class AtatWebApiStack extends cdk.Stack {
     // TODO: getApplicationStep
     // TODO: createApplicationStep
     // TODO: submitPortfolioDraft
-    // TODO: uploadTaskOrder
+    addCreateTaskOrderFiles(this, taskOrderFiles, sharedFunctionProps);
     // TODO: deleteTaskOrder
   }
 }
+
+function addCreateTaskOrderFiles(scope: cdk.Stack,
+                                 resource: apigw.Resource) {
+  const bucket = new s3.Bucket(scope, "PendingBucket", {
+    publicReadAccess: false,
+    removalPolicy: cdk.RemovalPolicy.RETAIN,
+    autoDeleteObjects: false
+  });
+  const fn = new lambdaNodejs.NodejsFunction(scope, "CreateTaskOrderFileFunction", {
```

Comment:
We may need to consider bumping the memory limit for this function to 256MB. I am concerned we're close enough to 128MB that if we actually did get a 10MB PDF that we'd OOM crash. Can you peek at the CWL data at the `REPORT` lines to see if we're within 10MB of 128MB of memory utilization?

Reply:
Probably too close for comfort. Here's one example for a 1.6 MB file: 

```
Duration: 1389.71 ms Billed Duration: 1390 ms Memory Size: 128 MB Max Memory Used: 106 MB Init Duration: 328.10 ms
```

I'll go ahead and raise it to 256MB.